lib/scalarm/database/core/mongo_active_record_utils.rb

```diff
@@ -41,6 +41,16 @@ def parse_json_if_string(attribute)
           end
         end
 
+        def create_index(attribute)
+          @_indexed_attributes << attribute
+          # puts "Create index: #{attribute} - #{@_indexed_attributes}"
+        end
+
+        def inherited(subclass)
```

Comment:
Jest to nadpisanie istniejacej metody rubiego - http://ruby-doc.org/core-2.2.0/Class.html#method-i-inherited . Tworzy zmienna instancyjna w podklasie z wartoscia domyslna.